src/scripts/main.js

```diff
@@ -1,17 +1,42 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const mainElement = document.querySelector('body');
+  const block = document.createElement('div');
+
+  block.classList.add('notification');
+  block.classList.add(`${type}`);
+  block.style.top = `${posTop}px`;
+  block.style.right = `${posRight}px`;
+  mainElement.append(block);
+
+  const h2 = document.createElement('h2');
+
+  h2.classList.add('title');
+  h2.innerText = title;
+  h2.style.whiteSpace = 'nowrap';
+  block.append(h2);
+
+  const p = document.createElement('p');
+
+  p.innerText = description;
+  block.append(p);
```

Comment:
```suggestion
   block.insertAdjacentHTML('beforeend', `
      <h2 class="title">${title}</h2>
      <p>${description}</p>
  `);
```
it can be shorter and more readable